dev-cpp/tbb/tbb-2018.3.recipe

```diff
@@ -4,13 +4,13 @@ developed by Intel for parallel programming on multi-core processors. Using \
 TBB, a computation is broken down into tasks that can run in parallel. The \
 library manages and schedules threads to execute these tasks."
 HOMEPAGE="https://www.threadingbuildingblocks.org/"
-COPYRIGHT="2005-2017 Intel Corporation"
+COPYRIGHT="2005-2018 Intel Corporation"
 LICENSE="Apache v2"
 REVISION="1"
-SOURCE_URI="https://github.com/01org/tbb/archive/2018_U2.tar.gz"
-CHECKSUM_SHA256="78bb9bae474736d213342f01fe1a6d00c6939d5c75b367e2e43e7bf29a6d8eca"
-PATCHES="tbb-2018_U2.patchset"
-SOURCE_DIR="tbb-2018_U2"
+SOURCE_URI="https://github.com/01org/tbb/archive/2018_U3.tar.gz"
+CHECKSUM_SHA256="23793c8645480148e9559df96b386b780f92194c80120acce79fcdaae0d81f45"
+PATCHES="tbb-2018_U3.patchset"
+SOURCE_DIR="tbb-2018_U3"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="https://github.com/01org/tbb/archive/2018_U3.tar.gz"
+SOURCE_URI="https://github.com/01org/tbb/archive/${portVersion/./_U}.tar.gz"
 CHECKSUM_SHA256="23793c8645480148e9559df96b386b780f92194c80120acce79fcdaae0d81f45"
-PATCHES="tbb-2018_U3.patchset"
-SOURCE_DIR="tbb-2018_U3"
+SOURCE_DIR="tbb-${portVersion/./_U}"
+PATCHES="tbb-${portVersion/./_U}.patchset"
```
Notice that `SOURCE_DIR` should be above `PATCHES`, according to: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)